app/views/boards/show.html.erb

```diff
@@ -53,3 +51,6 @@
   users: @users
 })
 %>
+<% if @board.previous_boards.any? %>
```

Comment:
Fantastically bad idea
If you need to know it there are parent boards you just can take a look at @board.previous_board_id
if its not empty, then there is a story of at least two boards